src/symbols.c

```diff
@@ -2289,6 +2290,32 @@ static gint get_fold_header_after(ScintillaObject *sci, gint line)
 }
 
 
+/* returns the line after following all brace match for @brace on @line */
+static gint resolve_matching_braces(ScintillaObject *sci, gint line, gint brace)
+{
+	gint pos = sci_get_position_from_line(sci, line);
+	gint line_end = sci_get_line_end_position(sci, line);
+	gint lexer = sci_get_lexer(sci);
```

Comment:
@b4n, agree that them being const is correct, and assists the compiler (and human) in understanding your intention.  Its not possible to do that in the initialiser if `pos` is non-const, so better out of it.